### PR TITLE
Add some changes to candidate profile

### PR DESCRIPTION
**Pull request by heavensbee001**

### CODE REVIEW

File: packages/ui/src/info/CandidateInfo/tabs/InfoTab.tsx [hunk 1]
```diff
@@ -45,34 +45,39 @@ export const InfoTab = ({ member }: Props) => {
...
+                  member?.budget?.perHour !== undefined &&
+                  member?.budget?.perHour >= 0 && (
+                    <p className="text-center text-sm">
+                      $USD{" "}
+                      <span className="text-xl font-bold text-[#fcba03]">
+                        {member?.budget?.perHour}
+                      </span>{" "}
+                      hour
+                    </p>
+                  )}
```
Comment (Markkos89) [inline packages/ui/src/info/CandidateInfo/tabs/InfoTab.tsx:80]: actually validating the variable as:

`member?.budget?.perHour`

is the same as all others

Comment (Markkos89) [reply packages/ui/src/info/CandidateInfo/tabs/InfoTab.tsx:80]: also, the conditional rendering with `&&` can lead to react printing a "0" in the screen if something goes bad.. is better to use ternaries imho

**Review by Markkos89 — CHANGES_REQUESTED** [review]
awesome fix! just 2 comments regarding conditional rendering and validation

File: packages/ui/src/info/CandidateInfo/tabs/InfoTab.tsx (hunk 1 above)
Comment (heavensbee001) [reply packages/ui/src/info/CandidateInfo/tabs/InfoTab.tsx]: Cool, submitted a fix on this :)

**Review by Markkos89 — APPROVED** [review]

**Pull request merged**